### PR TITLE
WIP? Rename "path" to "path start"

### PR DESCRIPTION
**Pull request by Scotchester**
To make it a bit clearer in the UI that the path condition matches on `startswith`.

This PR probably needs more work, and I want to make sure @willbarton gets a look at it, so I wanted to go ahead and open it so I don't forget to do it after he's back from his vacation.

### CODE REVIEW

**Comment by anselmbradford** [comment]
You could consider `path begins with` too, unless that's too wordy. 

screen shot 2017-07-05 at 4 18 25 pm

screen shot 2017-07-05 at 4 17 56 pm

**Comment by chosak** [comment]
How would you feel about instead modifying how this condition works so that it takes a regular expression?

It'd be a little harder for non-technical folks to set up, but perhaps there'd be some way to add help text to the interface. That way you could keep the current "starts with" behavior by doing something like `^/eregs/` but also do "exact match" behavior by doing something like `^/eregs/$`, not to mention all sorts of other fancy pattern matching.

**Comment by Scotchester** [comment]
I would certainly like to have the option to use regex!

**Comment by willbarton** [comment]
@chosak I like the regular expression idea, and I was actually working on a helper-text ability before I was out. I'll work on that.

In the meantime, @Scotchester shall we add regex support in this PR and change it to "path matches"?

**Comment by Scotchester** [comment]
Sounds great! I'll work on modifying the PR.

**Comment by Scotchester** [comment]
Closing in favor of #20.
